lib/coins/ada.js

```diff
@@ -0,0 +1,41 @@
+const b58 = require('b58')
+const cbor = require('cbor-sync')
+const CRC32 = require('crc-32'); 
+
+module.exports = {depositUrl, parseUrl}
+
+function parseUrl(network, address) {
+  console.log('DEBUG16: [%s] *%s*', network, address)
```

Comment:
I suppose this should be removed.

Reply:
This log is in all other coins (except Ethereum) - so are you sure about remove this?

Reply:
Ok in that case leave it here